README.md

```diff
@@ -1,11 +1,14 @@
 # Metal
 
-A programming language. We'll add a catchy slogan soon, pinky swear
+Prototype compiled language built in Rust with LLVM. We'll add a catchy slogan later.
+
+## Contributing
+
+Currently, Metal is more or less under source availability. Metal will eventually
+become fully open source, but for now, it's far too undeveloped to require an open source license.
```

Comment:
The "too underdeveloped to require an OSS licence" part sounds strange to me. No matter the degree of "developedness" of a piece of software, there's no point at which it requires to be OSS, right? Also, i think "source availability" suggests more freedom than absence of a license means (if not for GitHub's ToS, no one could even legally *view* the code). I propose leaving out this section until we decide on a license